clearblade.md

```diff
@@ -567,18 +567,18 @@ Sets data in the cache. Requires that the Cache object was initialized with a ca
 ~~~~
 
 ## Cache.setnx(key, value, callback)
-Sets data in the cache if it does not exist. Requires that the Cache object was initialized with a cache name. On success, this returns a boolean string. Returns "true" if it set the data in the cache. Returns "false" if the data already exists
+Sets data in the cache if it does not exist. Requires that the Cache object was initialized with a cache name. On success, this returns a boolean string. Returns boolean __true__ if it set the data in the cache. Returns boolean __false__ if the data already exists
 
 * @param {string} key - Key in cache
-* @param {string|number|object|array} value - Data to be stored in the cache
+* @param {string|number|object|array|boolean|null} value - Data to be stored in the cache
 * @param {function} callback - callback that returns error or success messages
```

Comment:
Do you mind adding separate jsdoc for the callback function, it's better than writing it in sentence. Also, because it's different than regular callbacks in clearblade

Reply:
Do you mind adding jsdoc for all the callbacks for cache.<method>, where ever it's different than standard clearblade one?

Reply:
what do you mean? Each callback is gonna return a different value depending on what people have set in their cache. How can we account for callbacks for each data type?

Reply:
I have examples in there that explain how to use a callback. I think people can figure it out from that

Reply:
I agree w Sanket. it is simple, and explained in the function description. Since October, the docs have been incorrect until this gets merged